CHANGELOG.md

```diff
@@ -19,6 +19,10 @@
 * `Profile` has increased margin between the image and text.
 * `Dialog` bottom padding has increased by 8px.
 
+## Bug Fixes
+
+* Fixes `Rainbow` widget Invoices graph issue where link to Invoices Summary did not work and threw an error (Sage One Start only). 
```

Comment:
I'd keep the release notes more generic to the fix, rather than explaining about in product bugs, as this repo is public and open source.